servers/rendering/renderer_rd/forward_clustered/render_forward_clustered.cpp

```diff
@@ -1082,6 +1085,12 @@ void RenderForwardClustered::_fill_instance_data(RenderListType p_render_list, i
 		instance_data.lightmap_uv_scale[2] = inst->lightmap_uv_scale.size.x;
 		instance_data.lightmap_uv_scale[3] = inst->lightmap_uv_scale.size.y;
 
+#ifdef DEBUG_ENABLED
+		if (unlikely(get_debug_draw_mode() == RS::VIEWPORT_DEBUG_DRAW_UV2_TEXEL_DENSITY && inst->data->use_baked_light)) {
```

Comment:
Should check if ``inst->data->lightmap_size_global_uniform_pos`` is valid
```suggestion
		if (unlikely(get_debug_draw_mode() == RS::VIEWPORT_DEBUG_DRAW_UV2_TEXEL_DENSITY && inst->data->use_baked_light && inst->data->lightmap_size_global_uniform_pos >=0)) {
```